src/include/cart/api.h

```diff
@@ -407,6 +407,34 @@ crt_reply_get(crt_rpc_t *rpc)
 	return rpc->cr_output;
 }
 
+/**
+ * Return HLC timestamp of RPC was sent
+ *
+ * \param[in] req              pointer to RPC request
+ *
+ * \return                     HLC timestamp
+ */
+uint64_t
+crt_hlc_send_get(crt_rpc_t *req);
+
+/**
+ * Return HLC timestamp of RPC (new or reply) was received
```

Comment:
What does "received" mean in this context?  Is this only to be called on the Tx side and received is when the reply was received on the node or is it something else?

Reply:
done